BoxContentSDK/BoxContentSDK/Requests/BOXRequest.m

```diff
@@ -78,6 +78,13 @@ - (BOXAPIOperation *)createOperation
     return nil;
 }
 
+
+- (void)prepareOperation
+{
+    [self.operation.APIRequest setValue:[self userAgent] forHTTPHeaderField:@"User-Agent"];
+    [self.operation prepareOperation];
```

Comment:
I don't think we need to... prepareOperation is exposed on request, and sessionTask is exposed on the operation that is exposed by the request.
Expected usage is like this:
           let request = BOXContentClient.default().fileUploadNewVersionRequestInBackground(withFileID:...)
            request?.requestAllFileFields = true
            request?.prepareOperation()
            if let task = request?.operation.sessionTask {
                NSFileProviderManager.default.register(task, ...